src/sdk/resources/transferItems.ts

```diff
@@ -16,11 +16,23 @@ export class TransferItemsResource extends PagedResource<TransferItem> {
 		return this.modifyItem(itemid, {retry: true}, token);
 	}
 
+	ignoreMultiple(itemids?: any, token?: CancellationToken): Promise<void> {
```

Comment:
Rename this to `params` since it wouldn't actually be a list of item IDs but rather our normal search parameters.